config/enabled.yml

```diff
@@ -948,6 +948,11 @@ Lint/SpaceBeforeFirstArg:
                  in a method call without parentheses.
   Enabled: true
 
+Lint/SpaceInsideStringInterpolation:
+  Description: 'No spaces inside "#{v}".'
```

Comment:
Maybe "Checks for padding/surrounding spaces inside string interpolation." would be better?